packages/components/src/components/drawer/drawer.ts

```diff
@@ -308,7 +318,7 @@ export default class SdDrawer extends SolidElement {
           )}
           role="dialog"
           aria-modal="true"
-          aria-hidden=${this.open ? 'false' : 'true'}
+          aria-hidden=${!this.open}
```

Comment:
You have to explicitly set the string: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden

Reply:
Updated